pkg/batchd/cache/resource_info.go

```diff
@@ -54,13 +61,16 @@ func NewResource(rl v1.ResourceList) *Resource {
 			r.MilliCPU += float64(rQuant.MilliValue())
 		case v1.ResourceMemory:
 			r.Memory += float64(rQuant.Value())
+		case GPUResourceName:
+			q, _ := rQuant.AsInt64()
+			r.GPU += q
 		}
 	}
 	return r
 }
 
 func (r *Resource) IsEmpty() bool {
-	return r.MilliCPU < minMilliCPU && r.Memory < minMemory
+	return r.MilliCPU < minMilliCPU && r.Memory < minMemory && r.GPU == 0
```

Comment:
What if GPU is not present on any node in the cluster?

Reply:
Pods won't be binded forever.

Reply:
Is that the desired behavior? Will all pod require GPU?

Reply:
> Is that the desired behavior? 

Probably yes.

> Will all pod require GPU?

It depends on workloads.